plugin/output/splunk/splunk.go

```diff
@@ -154,18 +155,22 @@ func (p *Plugin) out(workerData *pipeline.WorkerData, batch *pipeline.Batch) {
 	insaneJSON.Release(root)
 	data.outBuf = outBuf
 
+	p.logger.Debugf("Trying to send: %s", outBuf)
+
 	for {
 		err := p.send(outBuf)
 		if err != nil {
 			stats.GetCounter(subsystemName, sendErrorCounter).Inc()
-			p.logger.Errorf("can't send data to splunk address=%s: %s", p.config.Endpoint, err.Error())
+			p.logger.Errorf("Can't send data to splunk address=%s: %s", p.config.Endpoint, err.Error())
+			p.logger.Debugf("Error sending: %s", outBuf)
```

Comment:
@vano144 described why this is a bad idea: infinite logs if splunk went down.  `Trying to send` is enough. So this `Debugf` is rendundant.

Reply:
I thought this is what you said and it was a good idea

What infinite loop, could you explain more. This is logging of file.d instance

Reply:
@vano144 described that if splunk went down `for` can iterates for very long time, with same debug message which is useless. That's why he suggests write debug msg once before `for` and once after.